tests/UI/campaigns/functional/API/02_endpoints/01_apiAccess/01_getAPIApiAccessId.ts

```diff
@@ -0,0 +1,140 @@
+
+      accessToken = jsonResponse.access_token;
+    }
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  // Pre-condition: Enable experimental feature : Authorization server
+  setFeatureFlag(featureFlagPage.featureFlagAuthorizationServer, true, `${baseContext}_enableAuthorizationServer`);
+
+  // Pre-condition: Create an API Access
+  createAPIAccessTest(createAPIAccess, `${baseContext}_preTest`);
+
+  describe('BackOffice : Expected data', async () => {
+    it('should login in BO', async function () {
+      await loginCommon.loginBO(this, page);
+    });
+
+    it('should go to \'Advanced Parameters > API Access\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToAuthorizationServerPage', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.advancedParametersLink,
+        dashboardPage.authorizationServerLink,
+      );
+
+      const pageTitle = await apiAccessPage.getPageTitle(page);
+      expect(pageTitle).to.eq(apiAccessPage.pageTitle);
+    });
+
+    it('should get informations', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'getInformations', baseContext);
+
+      idApiAccess = parseInt(await apiAccessPage.getTextColumn(page, 'id_api_access', 1), 10);
+      expect(idApiAccess).to.be.gt(0);
+    });
+  });
+
+  describe('API : Check Data', async () => {
+    it('should request the endpoint /admin-dev/api/api-access/{apiAccessId}', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'requestEndpoint', baseContext);
+
+      const apiResponse = await apiContext.get(`api/api-access/${idApiAccess}`, {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
+      });
+      expect(apiResponse.status()).to.eq(200);
+      expect(api.hasResponseHeader(apiResponse, 'Content-Type')).to.eq(true);
+      expect(api.getResponseHeader(apiResponse, 'Content-Type')).to.contains('application/json');
+
+      const jsonResponse = await apiResponse.json();
+      // apiAccessId
+      expect(jsonResponse).to.have.property('apiAccessId');
+      expect(jsonResponse.apiAccessId).to.be.a('number');
+      expect(jsonResponse.apiAccessId).to.be.equal(idApiAccess);
+
+      // clientName
+      expect(jsonResponse).to.have.property('clientName');
+      expect(jsonResponse.clientName).to.be.a('string');
+      expect(jsonResponse.clientName).to.be.equal(createAPIAccess.clientName);
+
+      // description
+      expect(jsonResponse).to.have.property('description');
+      expect(jsonResponse.description).to.be.a('string');
+      expect(jsonResponse.description).to.be.equal(createAPIAccess.description);
+
+      // enabled
+      expect(jsonResponse).to.have.property('enabled');
+      expect(jsonResponse.enabled).to.be.a('boolean');
+      expect(jsonResponse.enabled).to.be.equal(createAPIAccess.enabled);
```

Comment:
why not split the expects on multiple steps ? it may be important to know what failed exactly by just reading the report